readme.md

```diff
@@ -94,6 +118,34 @@ const Employee = db.rawTable('Employee');
 
 ### Methods
 
+Every method can override the retry [options](https://github.com/tim-kos/node-retry#retryoperationoptions) passed with the `.connect()` method or can customise the retry configuration for the specific method.
+
+```js
+Employee
+	.find({Organisation: 'Amazon'})
+	.where({Salary: {$gt: 3000}})
+	.select('FirstName Name')
+	.retry({retries: 2})
+	.exec()
+    .then(employees => {
+        // => [{FirstName: 'Foo', Name: 'Bar'}]
+	});
+```
+
+If you don't want to configure the retry strategy, you can simply pass the number of retries.
+
+```js
+Employee
+	.find({Organisation: 'Amazon'})
+	.where({Salary: {$gt: 3000}})
+	.select('FirstName Name')
+	.retry(2)
+	.exec()
+    .then(employees => {
+        // => [{FirstName: 'Foo', Name: 'Bar'}]
+    });
```

Comment:
Indentation

Reply:
Could you give an suggestion on what the identation needs to be? It has the same identation as other documentation (see line 154)

Reply:
It looked off when I reviewed it. On mobile now and it looks ok.

Reply:
Same, it has spaces instead of tabs.